sn_client/src/wallet.rs

```diff
@@ -267,7 +347,8 @@ impl WalletClient {
     ///   `skipped_chunks` is the list of content already exists in network and no need to upload
     ///
     /// Note storage cost is _per record_, and it's zero if not required for this operation.
-    /// This can optionally verify the store has been successful (this will attempt to GET the cash_note from the network)
+    /// This can optionally verify the store has been successful.
+    /// This will attempt to GET the cash_note from the network
     pub async fn pay_for_storage(
         &mut self,
         content_addrs: impl Iterator<Item = NetworkAddress>,
```

Comment:
here `verify_store = true`, so I think we can mention that the verification is mandatory here.

Reply:
I have changed the line to this:
    /// * `verify_store` - A boolean to verify store. Set this to true for mandatory verification.
which helps to the use to understand that the verification is mandatory if set to true. 
Hope this helps :)